data/provider/src/main/java/tech/pegasys/artemis/api/ChainDataProvider.java

```diff
@@ -98,7 +99,8 @@ public boolean isStoreAvailable() {
     return combinedChainDataClient.getStateAtSlot(slot, root32);
   }
 
-  ChainStorageClient getChainStorageClient() {
+  @VisibleForTesting
+  public ChainStorageClient getChainStorageClient() {
     return chainStorageClient;
   }
```

Comment:
i really don't want this. Im planning on deleting these functions once we are done refactoring, so this will make that goal harder...